ADOPTERS.md

```diff
@@ -0,0 +1,43 @@
+## End-users
+
+Known end-users include: Bloomberg, Bytedance, Pinterest, Square, and Uber.
+
+SPIFFE and SPIRE are being used by numerous other companies, both large and small, to build higher layer products and services. This includes Decipher Technology Studios, F5 Networks, HashiCorp, Intel, Google, IBM, Tigera, VMware, and many others.
+
+## Ecosystem
+
+SPIFFE and SPIRE have integrations available with a number of open-source projects. The list includes:
+
+Kubernetes
+Docker
+Envoy
+Istio
+NGINX
+Ghosttunnel
+Knox	
+Consul
+Hamlet
+
+## Case Studies/User Stories
+
+Square talks about how Square uses SPIFFE and SPIRE to secure communications across hybrid infrastructure services:
+https://youtu.be/H5IlmYmEDKk?t=2585
+
+Uber talks about integrating SPIRE with workload schedulers: 
+https://youtu.be/H5IlmYmEDKk?t=4703
+
+Tigera demonstrates how Calico, Envoy and SPIRE are used to deliver unified Layer 4 and Layer 7 authorization policies:
+https://youtu.be/H5IlmYmEDKk?t=7812
+
+Bloomberg talks about TPM node attestation with SPIRE:
+https://youtu.be/30S0sKRxzjM
+
+NGINX/F5 on how NGINX service mesh leverages SPIFFE and SPIRE
+https://youtu.be/plRkDK5xFpM
+
+## Adding a name or logo
+
+If you would like to add your name or logo to this file, add a PNG version of your logo to the docs/adopters directory in this repo and submit a pull request with your change. Name the image file something that reflects your company (e.g., if your company is called Acme, name the image acme.png). 
```

Comment:
We should use a table for the Leading Adopters if we're going to have company logos on this page. Markdown tables are too limited to make nice-looking tables. I guess we could use HTML code to do it right.

Reply:
Sure. Done is better than perfect.